package.json

```
@@ -25,7 +25,8 @@
         "install:showcase-android": "cd demos/showcase && yarn && cd ../.. && yarn link:components",
         "install:storybook-ios": "cd demos/storybook && yarn && cd ios && pod install && cd ../../.. && yarn link:components",
         "install:storybook-android": "cd demos/storybook && yarn && cd ../.. && yarn link:components",
-        "install:storybook-api": "cd demos/api && yarn && cd ../.. && yarn link:components",
+        "install:storybook-api": "cd demos/api && yarn && cd ../.. && yarn link:components && copy-package-json",
```

```suggestion
        "install:storybook-api": "cd demos/api && yarn && cd ../.. && yarn link:components && yarn copy-package-json",
```

The `yarn start:storybook-api` command is broken if you don't include "yarn" here